sdk/include/opentelemetry/sdk/common/attribute_utils.h

```diff
@@ -33,22 +33,44 @@ namespace common
  */
 using OwnedAttributeValue = nostd::variant<bool,
                                            int32_t,
-                                           uint32_t,
                                            int64_t,
+                                           uint32_t,
                                            uint64_t,
                                            double,
                                            std::string,
+// TODO: should we have support for C-strings here or std::string is sufficient?
 #ifdef HAVE_SPAN_BYTE
                                            std::vector<uint8_t>,
 #endif
                                            std::vector<bool>,
                                            std::vector<int32_t>,
-                                           std::vector<uint32_t>,
                                            std::vector<int64_t>,
+                                           std::vector<uint32_t>,
                                            std::vector<uint64_t>,
                                            std::vector<double>,
                                            std::vector<std::string>>;
 
+enum OwnedAttributeType
```

Comment:
Let's follow the Google style guide for enumeration value names, to be consistent across the project.

Reply:
Ok